modules/configuration-annotation-processor/src/test/java/org/apache/ignite/configuration/notifications/ConfigurationListenerTest.java

```diff
@@ -70,9 +71,11 @@
     public void before() {
         registry.registerRootKey(ParentConfiguration.KEY);
 
-        registry.registerStorage(new TestConfigurationStorage());
+        TestConfigurationStorage storage = new TestConfigurationStorage();
 
-        registry.startStorageConfigurations(TestConfigurationStorage.class);
+        registry.registerStorage(storage);
```

Comment:
why do `registerStorage` and `startStorageConfigurations` have to be two separate methods?

Reply:
I can't really say, lets wait for @ibessonov